src/metric/EvaluationEmbedding.py

```diff
@@ -406,6 +410,90 @@ def train(dataset: Dataset, start_state: ModelUtil.AuxModelInfo = None) -> None:
     empty_cache()
 
 
+#! Projecting real samples
+
+
+def _ds_proj_file_name(dataset: Dataset) -> str:
+    return f"{DS_PROJ_FILE_PREFIX}_{dataset.get_name(dataset.get_resolution())}"
+
+
+def project(dataset: Dataset) -> None:
```

Comment:
Couldnt project return the projection instead of None. With this change you could sometimes skip unecessary I/O through project() ->  p = get_projections() and instead just write p = project()?